docs/README.md

```diff
@@ -1,24 +1,46 @@
 # Compiling Membrane Curvature's Documentation
 
-The docs for this project are built with [Sphinx](http://www.sphinx-doc.org/en/master/).
-To compile the docs, first ensure that Sphinx and the ReadTheDocs theme are installed.
+The docs for this project are built with [Sphinx](http://www.sphinx-doc.org/en/master/) using the [mdanalysis-sphinx-theme](https://github.com/MDAnalysis/mdanalysis-sphinx-theme).
+To compile the docs, first ensure that:
 
+1. All necessary dependencies are installed.
+1. The editable version of this repo is installed.
+1. You are working in the `/docs` directory.
+
+## Installing dependencies
+
+### Via conda
+
+To install a new environment with conda to compile the documentation, run:
 
 ```bash
-conda install sphinx sphinx_rtd_theme 
+conda install sphinx mdanalysis-sphinx-theme 
 ```
 
+### Via pip
+
+Alternatively, to install all doc dependencies in your environment run:
 
-Once installed, you can use the `Makefile` in this directory to compile static HTML pages by
 ```bash
-make html
+pip install -r requirements.txt
 ```
 
-The compiled docs will be in the `_build` directory and can be viewed by opening `index.html` (which may itself 
-be inside a directory called `html/` depending on what version of Sphinx is installed).
+## Installing the editable version
 
+Please souble check you have installed the editabe version of the package. **From the root of the project**, run:
```

Comment:
```suggestion
Please double check you have installed the editable version of the package. **From the root of the project**, run:
```

Although again as noted above, this may not be essential.